requirements/conda-3.4.yml

```diff
@@ -13,6 +13,7 @@ dependencies:
 - pytest=2.9.2=py34_0
 - python=3.4.5=0
 - python-dateutil=2.5.3=py34_0
+- python-vxi11=0.9=py34_0
```

Comment:
As far as I can tell, this package is only available in pip (https://pypi.org/project/python-vxi11/). Therefore this should go in the `pip` section at the bottom. Please make the change for all of the conda environment files.

Reply:
We should probably think about creating a conda forge feedstock for python-xvi11, otherwise pymeasure ceases to be installable via conda. maybe its maintainer is also interested in this?